server/routes/api/articles.js

```diff
@@ -0,0 +1,14 @@
+import ArticleController from '../../controllers/article';
+import auth from '../../middleware/auth';
+
+// get authenticateUser method
+const { authenticateUser } = auth;
+const router = require('express').Router();
+
+router.post('/', authenticateUser, ArticleController.create);
```

Comment:
Yes, I didn't add this because the tests will fail as no authorised author (user) has been created. It should be handled when implementing user  and roles and permissions